src/rgw/rgw_qos.h

```diff
@@ -0,0 +1,238 @@
+#pragma once
+#include <chrono>
+#include <thread>
+#include <condition_variable>
+#include "rgw_common.h"
+
+
+class RateLimiter {
+  static constexpr uint64_t fixed_float_rgw_ratelimit = 1000;
+  struct counters {
+    int64_t ops = 0;
+    int64_t bw = 0;
```

Comment:
please comment that the counters are tracked in multiples of `fixed_float_rgw_ratelimit`